xml/System.Security.Cryptography/CryptographicOperations.xml

```diff
@@ -89,9 +89,19 @@ Fixed-time behavior is guaranteed in all other cases, including when `left` and
         <Parameter Name="buffer" Type="System.Span&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="buffer">The buffer to fill with zeros.</param>
+        <summary>Fills the provided buffer with zeros.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method exists to future-proof against potential optimizations in the .NET runtime which could eliminate memory writes that are provably not followed by memory reads,
```

Comment:
nit - small typo:
```suggestion
This method exists to future-proof against potential optimizations in the .NET runtime which could eliminate memory writes that are probably not followed by memory reads,
```

Reply:
I've added a small change to the suggestion above as well

which -> that

Reply:
I meant "provably" (able to be proven), not "probably" (may or may not happen)

Reply:
Oh interesting. I didn't know that word existed. Thank you, native English speaker 😄 . Feel free to apply @mairaw 's suggestion alone if you want, @bartonjs.